client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -571,7 +571,15 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
             fetchTableData();
         }
         setIsBadgeInVisible(!Boolean(Object.values(filterRules).find(item => item !== null)))
-    }, [isLoggedIn, filterRules, orderBy, currentPage, displayedCounty, userType]);
+    }, [isLoggedIn, filterRules, orderBy, currentPage]);
+
+    useEffect(() => {
+        setCurrentPage(defaultPage);
+        if (isLoggedIn) {
```

Comment:
Just looked inside fetchTableData - and isLoggedIn is already in it, is it neccecary to check here also?

Reply:
removed the if